src/agent/Makefile

```diff
@@ -43,6 +43,11 @@ endif
 
 
 EXTRA_RUSTFLAGS :=
+
+ifeq ($(LIBC), gnu)
+    override EXTRA_RUSTFLAGS = -C target-feature=+crt-static
+endif
+
 ifeq ($(ARCH), aarch64)
     override EXTRA_RUSTFLAGS = -C link-arg=-lgcc
```

Comment:
@fidencio doesn't this mean that if someone were to build with glibc on aarch64, they'd miss out on static linkage? (Not that that's the most common thing, but still) I think you need `EXTRA_RUSTFLAGS += -C link-arg=-lgcc`

Reply:
Indeed, let me change this, thanks for the review!